testdata/goldens/template_testify.go

```diff
@@ -0,0 +1,15 @@
+package testdata
+
+import "testing"
+
+func TestFoo4(t *testing.T) {
+	tests := []struct {
+		name string
+		want bool
+	}{
+		// TODO: Add test cases.
+	}
+	for _, tt := range tests {
```

Comment:
Don't you want to use the `name` here somehow?

Reply:
Ahh... okay. I will try to work on that. Thank you.